src/annotator/guest.js

```diff
@@ -535,53 +535,46 @@ export default class Guest extends Delegator {
    * @param {Partial<AnnotationData>} annotation - Initial properties of the
    *   new annotation, other than `target`, `document` and `uri` which are
    *   set by this method.
+   * @return {Promise<AnnotationData>} - The updated annotation with selectors
+   *   and metadata populated.
```

Comment:
This method used to return an `AnnotationData` despite being async internally. It now returns a Promise as it should.